plasma_framework/contracts/src/exits/IStateTransitionVerifier.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.5.0;
+pragma experimental ABIEncoderV2;
+
+interface IStateTransitionVerifier {
```

Comment:
I would put this under `.../src/exits/interfaces/IStateTransitionVerifier.sol`. I think we will have a few interfaces for to be put there: `IOutputGuardHandler` (#227 ), `ISpendingCondition`(#214 ) and this one (and potentially more).

Reply:
done